features/full_tests/detect_anr_cxx.feature

```diff
@@ -11,15 +11,18 @@ Scenario: ANR triggered in CXX code is captured
     And the request is valid for the error reporting API version "4.0" for the "Android Bugsnag Notifier" notifier
```

Comment:
These scenarios need moving to the `batch_1` directory as currently CI isn't running them as tests.